app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -121,8 +121,8 @@ def check_enrollment(enrollment)
     enrollment.update(status_check_attempted_at: status_check_attempted_at)
   end
 
-  def passed_with_unsupported_secondary_id_type?(response)
-    return response['secondaryIdType'].present? &&
+  def passed_with_unsupported_secondary_id_type?(enrollment, response)
+    return !enrollment.enhanced_ipp? && response['secondaryIdType'].present? &&
```

Comment:
I am wondering if it makes sense to create a new method for create a separation between EIPP checks and ID IPP checks

```ruby
def has_unsupported_secondary_id_type?(response)
  response['secondaryIdType'].present? && 
    SUPPORTED_SECONDARY_ID_TYPES.exclude(response['secondaryIdType'])
end
```
Then the `passed_with_unsupported_secondary_id_type?` would look like this

```ruby
def passed_with_unsupported_secondary_id_type?(enrollment, response)
  !enrollment.enhanced_ipp? && has_unsupported_secondary_id_type?(response)
end
```

Reply:
Ideally it would be nice if we had a wrapper for the proofing response then we could have a method in the response wrapper that can make the check.

Reply:
Yea, that's a good thought, I'm not sure. Since this is a temporary change I would expect the final solution to either not have this check at all or expand the list of supported secondary ids based on being in the enhanced_ipp flow. I'm just not sure where this isolation would be.